packages/ingest/src/consumerRateLimited.ts

```diff
@@ -0,0 +1,161 @@
+/* eslint-disable no-await-in-loop */
```

Comment:
This should be removed (I added it to get the local commit hooks to pass without cluttering pre-edit), apologies